model-server/src/test/kotlin/org/modelix/model/server/ReplicatedRepositoryTest.kt

```diff
@@ -437,9 +439,9 @@ class ReplicatedRepositoryTest {
             nonMergedVersions.remove(versionToMerge)
         }
 
-        for (i in 1..100) {
+        (1..100).forEach {
```

Comment:
We should probably keep the regular for loop to conform with the style guide.

Reply:
Right, the change was older than our discussion about the style guide. I am still confused about that rule ;-)

Reply:
Replaced with `repeat`, which is hopefully even more readable now.